src/components/AdjustVaultForm.tsx

```diff
@@ -43,84 +45,98 @@ const AdjustVaultForm = () => {
 
   const { collateralError, debtError } = useAtomValue(adjustVaultErrorsAtom);
 
+  const [isCloseVaultDialogOpen, setIsCloseVaultDialogOpen] = useState(false);
+
   return (
-    <div className="bg-white font-serif p-8 shadow-[0_40px_40px_-14px_rgba(116,116,116,0.25)] rounded-[20px] w-full">
-      <div className="font-bold mb-4">Adjust Collateral</div>
-      <div className="mb-12 grid grid-cols-2 gap-10">
-        <div className="text-[#9193A5] text-sm col-span-2 lg:col-span-1">
-          Deposit additional collateral or withdraw your existing collateral.
-          Select &quot;No Action&quot; to leave collateral unchanged.
-        </div>
-        <div className="col-span-2 lg:col-span-1">
-          <div className="mb-6">
-            <StyledDropdown
-              selection={collateralAction}
-              choices={[
-                CollateralAction.None,
-                CollateralAction.Deposit,
-                CollateralAction.Withdraw,
-              ]}
-              onSelection={setCollateralAction}
-              label="Action"
+    <>
+      <div className="bg-white font-serif p-8 shadow-[0_40px_40px_-14px_rgba(116,116,116,0.25)] rounded-[20px] w-full">
+        <div className="font-bold mb-4">Adjust Collateral</div>
+        <div className="mb-12 grid grid-cols-2 gap-10">
+          <div className="text-[#9193A5] text-sm col-span-2 lg:col-span-1">
+            Deposit additional collateral or withdraw your existing collateral.
+            Select &quot;No Action&quot; to leave collateral unchanged.
```

Comment:
consider Unicode quote chars for readability,
```suggestion
            Select ‘No Action’ to leave collateral unchanged.
```